lib/Elastica/Query.php

```diff
@@ -356,8 +356,11 @@ public function addScriptField($name, AbstractScript $script)
      */
     public function addAggregation(AbstractAggregation $agg)
     {
-        if (!array_key_exists('aggs', $this->_params)) {
-            $this->_params['aggs'] = array();
+        /**
```

Comment:
Agree. Didn't spot this one. And We are quite a long way away from removing 5.5 support ;-)